packages/docs-app/src/examples/core-examples/buttonExamples.tsx

```diff
@@ -0,0 +1,175 @@
+/* !
+ * (c) Copyright 2024 Palantir Technologies Inc. All rights reserved.
+ */
+
+import * as React from "react";
+
+import { AnchorButton, Button, Icon, Tooltip } from "@blueprintjs/core";
+import { CodeExample, type ExampleProps } from "@blueprintjs/docs-theme";
+
+export const ButtonBasicExample: React.FC<ExampleProps> = props => {
+    const code = `<Button text="Click Me" />`;
```

Comment:
This seems okay for now, but would love to see if we could find a way to keep these in sync.

Reply:
Yep! Will continue to iterate here on an optimal way to define these